libsolidity/interface/StandardCompiler.cpp

```diff
@@ -462,8 +462,6 @@ boost::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompile
 	if (sources.empty())
 		return formatFatalError("JSONError", "No input sources specified.");
 
-	ret.errors = Json::arrayValue;
```

Comment:
Any reason to remove this?

Reply:
This was most likely a rebase error. Fixed in commit 1a7e09ab7